core/src/test/java/org/bitcoinj/core/FullBlockTestGenerator.java

```diff
@@ -759,7 +759,7 @@ public boolean add(Rule element) {
                     try {
                         ByteArrayOutputStream bos = new UnsafeByteArrayOutputStream(73);
                         bos.write(coinbaseOutKey.sign(hash).encodeToDER());
-                        bos.write(SigHash.SINGLE.ordinal() + 1);
```

Comment:
yep, definitely. both have the same result in this case. but, not using byteValue() avoids some unnecessary casting so i'll update this in my next commit. i only used byteValue() because i liked the readability.